plugins/modules/packaging/os/portage.py

```diff
@@ -228,22 +240,36 @@
 
 import os
 import re
+import sys
+import traceback
 
-from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.basic import AnsibleModule, missing_required_lib
+from ansible.module_utils.common.respawn import has_respawned, respawn_module
 from ansible.module_utils.common.text.converters import to_native
 
 
+try:
+    from portage.dbapi import vartree
+    from portage.exception import InvalidAtom
+    HAS_PORTAGE = True
```

Comment:
While it's not necessary, linters prefer if you assign it anyway ;)
```suggestion
    HAS_PORTAGE = True
    PORTAGE_IMPORT_ERROR = None
```

Reply:
Indeed! Just wasn't sure if there was a preference.